cmake/scripts/common/Macros.cmake

```diff
@@ -499,7 +499,7 @@ macro(core_add_optional_subdirs_from_filelist pattern)
       list(GET line 1 subdir_dest)
       list(GET line 3 opts)
       foreach(opt ${opts})
-        if(ENABLE_${opt})
+        if(ENABLE_${opt} OR ${opt}_FOUND)
```

Comment:
this looks wrong. It will cause existing  but disabled stuff to be used. Maybe AND was intended here?